### PR TITLE
NEPT-1026: Add custom patch designed for media 2.0-beta1

### PR DESCRIPTION
**Pull request by gillesdeudon**

### CODE REVIEW

File: resources/patches/media-inputs_whitelist-1026.patch [hunk 1]
```diff
@@ -0,0 +1,69 @@
+diff --git a/media.module b/media.module
+index 1f97bcb..1f4e995 100644
+--- a/media.module
++++ b/media.module
+@@ -818,6 +818,13 @@ function media_element_process($element, &$form_state, $form) {
+   // Media browser attach code.
+   $element['#attached']['js'][] = drupal_get_path('module', 'media') . '/js/media.js';
+ 
++  // Caching field settings ofr security reason.
```
Comment (jfhovinne) [inline resources/patches/media-inputs_whitelist-1026.patch]: Typo.

File: resources/patches/media-inputs_whitelist-1026.patch [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
++  $cid = drupal_get_token(drupal_random_bytes(32));
++  cache_set('media_options:' . $cid, $element['#media_options']['global'], 'cache_form', REQUEST_TIME + 21600);
++
++  // Add the security token to the media options sent with the upload form..
```
Comment (jfhovinne) [inline resources/patches/media-inputs_whitelist-1026.patch]: Typo.

**Comment by jfhovinne** [comment]
:+1:

**Comment by imanoleguskiza** [comment]
👍

**Pull request merged**